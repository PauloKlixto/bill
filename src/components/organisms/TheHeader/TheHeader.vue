<template>
  <section class="TheHeader">
    <header class="TheHeader__topBar">
      <p>Neal.fun</p>
    </header>
    <section class="TheHeader__billGatesArea">
      <img class="TheHeader__billPhoto" src="@/assets/images/billGates.png" />
      <h1>Spend Bill Gates' Money</h1>
    </section>
  </section>
</template>

<script>
export default {
  name: 'TheHeader'
}
</script>

<style lang="scss" scoped>
.TheHeader {
  margin-bottom: 7.5px;
  &__topBar {
    width: 100vw;
    background-color: $color-white;
    font-size: 30px;
    font-weight: bold;
    padding: 30px 15px;
    margin-bottom: 15px;
    p {
      max-width: 980px;
      width: 100vw;
      margin: auto;
    }
  }
  &__billGatesArea {
    max-width: 950px;
    height: 260px;
    width: 100vw;
    margin: auto;
    background: $color-white;
    display: grid;
    justify-items: center;
    gap: 20px;
    h1 {
      font-size: 32px;
      text-align: center;
    }
  }
  &__billPhoto {
    border-radius: 50%;
    align-self: flex-end;
  }
}
</style>
