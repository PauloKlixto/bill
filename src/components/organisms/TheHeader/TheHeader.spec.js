/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import TheHeader from './TheHeader'

describe('TheHeader', () => {
  const wrapper = shallowMount(TheHeader)

  test('Componente existe', () => {
    expect(wrapper.exists()).toBe(true)
  })

  test('Componente nÃ£o alterado', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
