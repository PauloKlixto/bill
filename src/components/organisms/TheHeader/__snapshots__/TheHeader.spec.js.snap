// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeader Componente n√£o alterado 1`] = `
"<section class=\\"TheHeader\\">
  <header class=\\"TheHeader__topBar\\">
    <p>Neal.fun</p>
  </header>
</section>"
`;
